### PR TITLE
Update fax number in site footer.

### PR DESCRIPTION
**Pull request by nader77**
#149 
selection_105

### CODE REVIEW

File: app/index.html [hunk 1]
```diff
@@ -325,7 +325,10 @@
           5 Brenner St., Tel-Aviv, Israel
         </div>
         <div>
-          <a href="tel:+97233731222" class="telephone">+972-3-3731222</a>
+          Tel: <a href="tel:+97233731222" class="telephone">+972-3-3731222</a>
+        </div>
+        <div>
+          Fax: <a href="tel:+972035617771" class="telephone">+972-3-5617771</a>
```
Comment (amitaibu) [inline app/index.html]: maybe remove the ``<a>`` & ``tel:``

**Comment by nader77** [comment]
@amitaibu Removed link tag from fax.

**Comment by amitaibu** [comment]
If font  color /size is looking ok, please merge

**Comment by nader77** [comment]
selection_106

Merging.

**Comment by nader77** [comment]
Merged.

**Pull request closed without merging**